cmd/dockerd/docker.go

```diff
@@ -78,6 +78,8 @@ func main() {
 		logrus.SetOutput(stderr)
 	}
 
+	initETWLogging()
```

Comment:
Given that there's already differences between Windows and Linux above; would it make sense to create a `initLogging` (e.g.) function, and move all Linux/Windows specific initialisation to implementations in a `_windows.go` and `_linux.go` respectively?

Reply:
Yes, that makes sense. Fixed in an update about to be pushed.